Dockerfile

```diff
@@ -31,11 +31,27 @@ RUN        true \
            && rm -rf /root/.npm \
            && true
 
+# Build a "files" layer
```

Comment:
not a fan of "files".  maybe a "src" layer?

Reply:
👍  used "source" in d4096c3c2e6c2165c678acb2557eba67ab5f3ff9